wbooks/src/app/screens/dashboard/screens/Home/components/SearchBar/styles.css

```diff
@@ -0,0 +1,31 @@
+.search-bar {
+  display: flex;
+  height: 40px;
+  align-items: center;
+  margin: 10px;
+
+}
+
+.search-bar select {
+  width: 250px;
+  height: 98%;
+  border: none;
+  margin: 0 15px 0 0;
+  background-color: #f4f4f4;
+}
+
+.search-bar input {
```

Comment:
`.search-bar input` and `.search-bar select` have common styles. Would you like to extract them ?